### PR TITLE
add findAccountForRemoteUrl tests

### PR DESCRIPTION
**Pull request by shiftkey**
Extracting these tests from #4362 for review while we dig into the right fix

### CODE REVIEW

File: app/test/unit/find-account-test.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      getEnterpriseAPIURL('https://github.mycompany.com'),
+      'deadbeef',
+      [],
+      '',
+      2,
+      'My Company'
+    ),
+  ]
+
+  it.skip('gives no account for non-GitHub endpoint', async () => {
```
Comment (iAmWillShepherd) [inline app/test/unit/find-account-test.ts]: This test shouldn't be skipped right? It looks like the one test that the original issue was about.

Comment (iAmWillShepherd) [reply app/test/unit/find-account-test.ts]: Alternatively a comment on why this test is being skipped will do.

Comment (shiftkey) [reply app/test/unit/find-account-test.ts]: Yeah, I'm skipping the test so that the build isn't broken. I'll add a comment.

**Comment by shiftkey** [comment]
Dropping the `ready-for-review` label to figure out why Travis has two failing tests and the others pass.

**Comment by shiftkey** [comment]
That seems to be working now? Uh, sure

**Comment by shiftkey** [comment]
> Just want to confirm we want to leave the test on line 30 marked as skipped.

@iAmWillShepherd yes, that's being tracked in #4154 and we don't want to break the build just yet.

**Pull request merged**